src/index.html

```diff
@@ -1,13 +1,343 @@
+                  <img
+                    src="/images/Charger.png"
+                    alt="Accessories"
+                    class="category__photo"
+                  >
+                </a>
+              </div>
+              <h2 class="category__title">Accessories</h2>
+            </article>
+            <a href="#categories" class="view-all">View all</a>
+          </div>
+        </section>
+      </div>
+
+      <div class="division-background"></div>
+
+      <section class="about-us" id="about-us">
+        <div class="container">
+          <div class="about-us__container">
+            <h2 class="section-title about-us__title">About us</h2>
+            <div class="about-us__text-container">
+              <p class="about-us__text">
+                We&#39;re a London-based tech company on a mission to remove barriers between people and technology.<br><br>
+                To do this, we&#39;re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+                No confusing tech-speak. No silly product names.Just artistry, passion and trust.
+                And products we&#39;re proud to share with our friends and family. Simple.<br><br>
+                Because beautiful tech shouldn&#39;t be complicated.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+    </main>
+    <div class="container">
+      <footer class="footer" id="footer">
+        <h2 class="section-title">Contact us</h2>
+        <div class="footer__container">
+          <form
+            class="footer__form form"
+            action="#"
+            method="post"
+            onsubmit="this.reset(); return false;"
+          >
+            <label class="form__label">
+              <input
+                type="text"
+                class="form__input"
+                placeholder="Name (e.g.,'Juanita')"
+                minlength="3"
+                maxlength="20"
+                required
+              >
+            </label>
+            
+            <label class="form__label">
+              <input
+                type="email"
+                class="form__input"
+                placeholder="Email (e.g.,'juanita@mail.mx')"
+                required
+              >
+            </label>
+
+            <label class="form__label">
+              <textarea
+                type="text" 
+                class="form__text-area"
+                placeholder="Message (e.g.,'Dear Nothing phone, how are you doing? What's up)"
+                maxlength="180"
+                required
+              ></textarea>
+            </label>
+          
+
+            <button
+              class="form__button button"
+              type="submit"
+            >
+              Send
+            </button>
+
+          </form>
+          <div class="footer__description">
+            <div class="footer__block">
+              <p class="footer__p">
```

Comment:
Better to give class names more meaningful names, for example `footer_block-title`.
Then you won't be dependent on using a specific tag for that class.